### PR TITLE
Sound card configurable. Localizable for zh-Hans.

### PR DESCRIPTION
**Pull request by js-john**
IMG_3944
IMG_3945

### CODE REVIEW

File: Configuration/UTMConfiguration.m [hunk 1]
```diff
@@ -1327,6 +1340,7 @@ - (id)initDefaults:(NSString *)name {
         self.networkEnabled = YES;
         self.printEnabled = YES;
         self.soundEnabled = YES;
+        self.soundCardDeviceId = @0;
```
Comment (osy86) [inline Configuration/UTMConfiguration.m]: For other values such as architectures and targets, we don't store the index but the actual value. Please do this as well. We don't want to force a mapping of id to value for future compatibility.

**Comment by osy86** [comment]
Please `git rebase` to master instead of merging master.

**Comment by js-john** [comment]
ok

**Comment by osy86** [comment]
The PR is good, you just need to make some changes.

**Pull request closed without merging**